Movie.h

```diff
@@ -0,0 +1,65 @@
+#include "Genre.h"
+#include <iostream>
+#include <string>
+#include <cstdlib>
+#include <ctime>
+#include <vector>
+using namespace std;
+class Movie
+{
+private:
+    type genre;
+    string title;
+    float id;
+    float ranking;
+    int day;
+    int month;
+    int year;
+    int characterNumber;
+
+    string comment;
+
+public:
+    float ticketPrice;
+    Movie(){
+        
+    };
+    Movie(float a, float b, string c, int d, int e, int f, int g, float h, string com, type tmpGenre)
+    {
+        id = a;
+        ranking = b;
+        title = c;
+        day = d;
+        month = e;
+        year = f;
+        characterNumber = g;
+        ticketPrice = h;
+        comment = com;
+        genre = tmpGenre;
+    }
+    ~Movie()
+    {
+    }
+
+    type getGenre()
+    {
+        return genre;
+    }
+    
+    float getRanking(){
```

Comment:
The name of method shoud be in uppercase.

Reply:
Can be as it is.